framework-cloud/framework-cloud-api/src/main/java/org/kie/cloud/api/deployment/WorkbenchDeployment.java

```diff
@@ -67,4 +67,19 @@
      * @return Workbench user password
      */
     String getPassword();
+
+    /**
+     * Change Kie Workbench user password.
+     *
+     * @param newPassword New workbench user password
+     */
+    void changePassword(String newPassword);
+
+    /**
+     * Change Kie Workbench user name and password.
+     *
+     * @param newUsername New workbench user name
+     * @param newPassword New workbench user password.
+     */
+    void changeUsernameAndPassword(String newUsername, String newPassword);
```

Comment:
I think this is a leftover. If we remove the changes in HttpsWorkbenchTestProvider, this is not used anywhere.